[INFO   ] Logger: Record log in /storage/emulated/0/code/util/Misc/kivy/kivy-examples/demo/kivycatalog/.kivy/logs/kivy_22-08-26_0.txt
[WARNING] [Config      ] Upgrading configuration in progress.
[WARNING] [Config      ] Older configuration version detected (0 instead of 21)
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  9 2021, 08:02:40) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 build 1.9@4893595'>
[INFO   ] GL: OpenGL vendor <b'Imagination Technologies'>
[INFO   ] GL: OpenGL renderer <b'PowerVR Rogue GE8300'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <4096>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[INFO   ] Video: Provider: ffmpeg
[WARNING] Image: Unable to load image </storage/emulated/0/code/util/Misc/kivy/kivy-examples/widgets/cityCC0.mpg>
[ERROR  ] Image: Error loading <../../widgets/cityCC0.mpg>
[INFO   ] GL: NPOT texture support is available
[WARNING] Base: Unknown <android> provider
[INFO   ] Base: Start application main loop
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/code/util/Misc/kivy/kivy-examples/demo/kivycatalog/main.py", line 192, in <module>
[WARNING] stderr:     KivyCatalogApp().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/app.py", line 950, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 582, in runTouchApp
[WARNING] stderr:     EventLoop.mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 347, in mainloop
[WARNING] stderr:     self.idle()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 387, in idle
[WARNING] stderr:     Clock.tick()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick
[WARNING] stderr:     self.post_idle(ts, self.idle())
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/clock.py", line 776, in post_idle
[WARNING] stderr:     self._process_events()
[WARNING] stderr:   File "kivy/_clock.pyx", line 616, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 649, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 645, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 218, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/video.py", line 191, in _do_video_load
[WARNING] stderr:     self._video.play()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/core/video/video_ffmpeg.py", line 47, in play
[WARNING] stderr:     self._player = ffmpeg.FFVideo(self._filename)
[WARNING] stderr: AttributeError: module 'ffmpeg' has no attribute 'FFVideo'
